Reasonable.md

```diff
@@ -0,0 +1,25 @@
+# Reasonable
+
+To be reasonable is to be ALL (conjuctive) of:
+* rational - to hold logically consistent beliefs and values
+* agnostic - to have no faith
+* corrigible - able to be corrected
+* fallible - capable of making mistakes
```

Comment:
Just an idea, we could take this further and say that all our beliefs must also be fallible. Or does that follow from saying that we ourselves are fallible?

Reply:
I would say the corresponding adjective for beliefs is "falsifiable".